File: Javascript/Arrays/Insercion_final.js
/*
Arrays en Javascript,Insercion en el ultimo elemento de un array.
@Autor: Daniel Alejandro Alvarado Velázquez
@grupo: #2-02
@Fecha: 03/09/2026
*/

//declaracion de arreglos
const input_arr = [11, 21, 31, 41, 51, 61];
const elem = 34;

console.log("Antes de la insercion del arreglo:");
for (let i = 0; i < input_arr.length; i++){
    console.log(input_arr[i]);
}

input_arr.push(elem);       //inserta un elemento al final de arreglo modificando su longitud

console.log("\nDespues de la insercion del arreglo: ");
for (let i = 0; i < input_arr.length; i++){
    console.log(input_arr[i]);
}

File: C#/Arrays/Insercion_final.cs
/*
Arrays en C#,Insercion en el ultimo elemento de un array.
@Autor: Daniel Alejandro Alvarado Velázquez
@grupo: #2-02
@Fecha: 06/09/2026
*/

using System;

namespace Isercion_final
{
    class Program
    {
        static void Main(String[] args)
        {
            int [] input_arr = {1, 11, 21, 31, 41, 51};
            int elem = 27;
            int tam = input_arr.Length;     //tamaño del arreglo

            Console.WriteLine("Antes de la insercion, el array es: ");
            foreach (int elemento in input_arr){
            Console.WriteLine(elemento);
            }
            
            Array.Resize(ref input_arr, tam);       //redimenciono el arreglo ya existente
            input_arr[tam - 1 ] = elem;

            Console.WriteLine("Despues de la insercion, el array es: ");
            foreach (int elemento in input_arr){
            Console.WriteLine(elemento);
            }

        }
    }
}